lib/engines/bh.js

```diff
@@ -7,37 +7,49 @@ module.exports = function (opts) {
 
     /**
      * bh.js express view engine adapter
-     * @api
-     * @returns {function(string, Object, Function)}
+     * Produces html from bh-templates and data and invokes passed callback with result
+     *
+     * @param   {String}     name       bundle name
+     * @param   {Object}     [options]  data to use in templates
+     * @param   {Function}   cb         callback to invoke with result or error
+     * @returns {Undefined}             undefined value
```

Comment:
Yes, because the function returns `undefined` in any case.

Previous declaration `@returns {function(string, Object, Function)}` was wrong. I hope you agree.